petrinet/scripts/julia/test.jl

```diff
@@ -0,0 +1,38 @@
+# Testing
+#########
+
+include("./ASKEMPetriNets.jl")
+using AlgebraicPetri
+using Catlab.CategoricalAlgebra, Catlab.Graphics
+using JSON
+
+# Load a model
+askemnet = ASKEMPetriNets.to_petri("../../examples/sir.json")
+
+askemnet.model |> to_graphviz
```

Comment:
I get this error from Line 12. Not a big deal, probably versioning of `Catlab.Graphics`
![Screenshot 2023-06-20 at 10 17 26](https://github.com/DARPA-ASKEM/Model-Representations/assets/1030720/27087900-948d-4d1f-9a3b-df26ff368e58)

Reply:
Yeah, I updated the Project.toml with compat entries to help you get the correct versions of things